widgets/src/lib/InferenceWidget/shared/consts.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Color palette for obj-det & img-seg widgets
+ */
+export const COLORS = [
+	{
+		"color": "red",
+		"hex": "#f87171",
+		"r": 248,
+		"g": 113,
+		"b": 113
+	},
```

Comment:
Great points. I've applied suggestion b
> set the rgb value from a function that would be parsing the hexadecimal functions

I've removed the rgb from the const (now it only has {color, hex}). The reason 

I did **not** apply suggestion a
> get the hex value directly from the tailwind theme

because I didn't want to add tailwind dependency to moon-landing/server